src/type/__tests__/validation-test.ts

```diff
@@ -365,7 +371,7 @@ describe('Type System: A Schema must have Object root types', () => {
       schema,
       parse(`
         extend schema {
-          subscription: SomeInputObject
+          subscription: SomeInputObject3
```

Comment:
Maybe we could change the other types to be something other than Input types while we were at it, not critical

Reply:
Done ✅